### PR TITLE
doc: Powering GNSE using Voltaic Solar Charger

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

GNSE can be powered using alternative sources of power, such as solar power, instead of relying on batteries. This guide explains how to use the Voltaic Lithium-Ion Capacitor Solar Charger to power the GNSE.

#### Screenshots

image

#### Changes

- ...
- ...

#### Notes for Reviewers

@azerimaker and @nejraselimovic 

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by azerimaker — DISMISSED** [review]
Nice work, @pradeeka7.

**Review by nejraselimovic — DISMISSED** [review]
Please fix these minor things and we can merge then. 

@azerimaker do you know if docs are being published automatically after merging to master branch now?

File: doc/content/getting-started/se-hands-on-guide/using-solar-power/_index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+description: ""
+weight:
+distributions: null
+---
+
+This tutorial will show you how to power the {{% gnse %}} using the [Voltaic Lithium-Ion Capacitor Solar Charger](https://voltaicsystems.com/LIC-solar-charger/), which includes a 250F Lithium-Ion Capacitor for power storage and a Linear regulator (C116). The solar charger outputs an unregulated 2.5V - 3.8V, which is sufficient for powering the {{% gnse %}} as it accepts a range of input voltages from 2.2V to 5.5V. 
+
+{{< figure src="voltaic-licap-solar-charger.jpg" alt="Voltaic Li-Ion Capacitor Solar Charger" >}}
+
+Additionally, you will need a [0.3 Watt 6 Volt Mini Solar Panel - ETFE](https://voltaicsystems.com/0-3-watt-6-volt-solar-panel-etfe/) to work with the solar charger.
```
[the remark this answers is not in the document]
Comment (pradeeka7) [reply doc/content/getting-started/se-hands-on-guide/using-solar-power/_index.md:12]: Thanks! @nejraselimovic for reviewing.

**Review by nejraselimovic — APPROVED** [review]

**Comment by pradeeka7** [comment]
@azerimaker, Could you please merge this?

**Comment by pradeeka7** [comment]
@nejraselimovic Could you please publish this page to the live site?

**Comment by nejraselimovic** [comment]
> @nejraselimovic Could you please publish this page to the live site?

no problem, will do

**Comment by nejraselimovic** [comment]
> @nejraselimovic Could you please publish this page to the live site?

I published these, can you pls check if it looks fine?

**Comment by pradeeka7** [comment]
> I published these, can you pls check if it looks fine?

I checked. It looks fine. Thanks @nejraselimovic

**Pull request merged**